### PR TITLE
[acl] Enable VLAN ID qualifier for ACL rules

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

**What I did**
I added support for the OUTER_VLAN_ID field to ACL rules.

**Why I did it**
To allow users to create rules that match on the VLAN ID of a packet.

**How I verified it**
See VS test updates. I also ran traffic tests against 201911 and master image.

**Details if related**

### CODE REVIEW

File: orchagent/aclorch.cpp [hunk 1]
```diff
@@ -288,6 +292,17 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
             value.aclfield.data.u16 = to_uint<uint16_t>(attr_value);
             value.aclfield.mask.u16 = 0xFFFF;
         }
+        else if (attr_name == MATCH_VLAN_ID)
+        {
+            value.aclfield.data.u16 = to_uint<uint16_t>(attr_value);
+            value.aclfield.mask.u16 = 0xFFF;
```
Comment (qiluo-msft) [inline orchagent/aclorch.cpp:298]: >0xFFF [](start = 38, length = 5)

0xFF ?

**Review by qiluo-msft — APPROVED** [review]

File: orchagent/aclorch.cpp [hunk 2]
```diff
@@ -1392,6 +1407,10 @@ bool AclTable::create()
         table_attrs.push_back(attr);
     }
 
+    attr.id = SAI_ACL_TABLE_ATTR_FIELD_OUTER_VLAN_ID;
```
Comment (prsunny) [inline orchagent/aclorch.cpp:1410]: So this is expected to be set for all tables except PFC_WD and ACL_TABLE_MIRROR_DSCP?

File: orchagent/aclorch.cpp (hunk 1 above)
Comment (prsunny) [inline orchagent/aclorch.cpp:298]: Should this be 0xFFFF?

Comment (qiluo-msft) [reply orchagent/aclorch.cpp:298]: 4095 == 0xFFF

File: orchagent/aclorch.cpp (hunk 2 above)
Comment (daall) [reply orchagent/aclorch.cpp:1410]: L3, L3V6, MIRROR, and MIRRORV6. DSCP and PFCWD are skipped in this method, and MCLAG and DTEL use their own init methods.

File: orchagent/aclorch.cpp (hunk 1 above)
Comment (daall) [reply orchagent/aclorch.cpp:298]: Exactly what @qiluo-msft said, VLAN ID is a 12 bit field so we need a 12 bit mask for it.

**Review by prsunny — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Comment by abdosi** [comment]
@daall please create pr for 201911. VS tests have conflict.

**Comment by daall** [comment]
> @daall please create pr for 201911. VS tests have conflict.
> 
> @qiluo-msft

ack, thx

**Pull request merged**